lib/utils/seedutil.ts

```diff
@@ -5,14 +5,14 @@ import { promisify } from 'util';
 const exec = promisify(childProcessExec);
 
 /**
- * Attempts to execute the seedutil tool which will generate an ethereum keystore
- * according to a given mnemonic and password at the specified path
+ * Executes the seedutil tool to generate an ethereum keystore from the given
+ * mnemonic and password at the specified path.
  * @param mnemonic the 24 seed recovery mnemonic
  * @param password the password to protect the keystore
  * @param path the path in which to create the keystore directory
  */
-const seedutil = async (mnemonic: string[], password: string, path: string) => {
-  const { stdout, stderr } = await exec(`./seedutil/seedutil -pass=${password} -path=${path} ${mnemonic.join(' ')}`);
+async function keystore(mnemonic: string[], password: string, path: string) {
```

Comment:
Why aren't we using arrow functions in files like this one again? I think Moshe told me a while ago but I forgot already :pensive:

Reply:
I think the main difference is that it's not a member method of a class, where usage of `this` can complicate things. In this case it's just a plain function that makes no use of `this`, whereas within class methods we frequently use `this` and always want it to refer to an instance of that class. @LePremierHomme feel free to chime in if you have any thoughts.